nimble/controller/include/controller/ble_ll.h

```diff
@@ -34,6 +35,14 @@ extern "C" {
 #error 32.768kHz clock required
 #endif
 
+#define BLE_LL_ASSERT(cond) \
+    if (!(cond)) { \
+        ble_ll_hci_ev_send_vendor_err(__FILE__, __LINE__); \
+        if (hal_debugger_connected()) { \
```

Comment:
this is not portable

Reply:
and why this is that?  It is HAL.

Reply:
we do not port HAL and LL should be independent of hw